zebedee-cms/src/main/java/com/github/onsdigital/zebedee/session/store/JWTStore.java

```diff
@@ -70,29 +99,75 @@ public Session get(String id) throws IOException {
      */
     @Override
     public Session get(HttpServletRequest id) throws IOException {
-        return get();
+        info().log(GET_REQUEST_ID_NOOP);
+        return null;
     }
 
     /**
      * Get a {@link Session} session object from thread local.
-     *
+     *  
      * @param none.
      * @return session object from thread local.
      * @throws IOException for any problem getting a session from the request.
      */
     @Override
-    public Session get() throws IOException {
-        return null;
+    public ThreadLocal<UserDataPayload> get() throws IOException {
```

Comment:
This needs to return a `Session`.

I am not 100% sure on how we do this yet. But for now I think:
- Get the value from the store.
- Construct a new Session and set the email field using the email in UserDataPayload.

We may need to revisit/refine this but for now that will be fine.